packages/flutter_tools/lib/src/asset.dart

```diff
@@ -203,6 +231,25 @@ class _Asset {
 
   @override
   String toString() => 'asset: $assetEntry';
+
+  @override
+  bool operator ==(dynamic other) {
+    if (identical(other, this))
+      return true;
+    if (other.runtimeType != runtimeType)
+      return false;
+    final _Asset otherAsset = other;
+    return base == otherAsset.base
```

Comment:
You can improve vertical alignment and consistency by always mentioning `other` first.